src/app/components/stacks-transaction-item/stacks-transaction-item.tsx

```diff
@@ -64,6 +67,22 @@ export function StacksTransactionItem({
     })();
   };
 
+  const onCancelTransaction = () => {
+    if (!transaction) return;
+    setRawTxId(transaction.tx_id);
+
+    const urlSearchParams = `?${createSearchParams({ txId: transaction.tx_id })}`;
+
+    whenWallet({
+      ledger: () =>
+        whenPageMode({
+          full: () => navigate(RouteUrls.CancelStxTransaction),
+          popup: () => openIndexPageInNewTab(RouteUrls.CancelStxTransaction, urlSearchParams),
+        })(),
+      software: () => navigate(RouteUrls.CancelStxTransaction),
+    })();
```

Comment:
> I think we can always just navigate? Not sure we need to handle ledger/software differently here.

I coped this from increase fee logic not sure why it was in that way since increase fee was working fine i thought it's better to rely on already established code